package.json

```diff
@@ -56,4 +56,4 @@
     "lodash.set": "^4.3.2",
     "lodash.debounce": "^4.0.8"
   }
-}
+}
```

Comment:
EOL :(

Reply:
I added a commit fe5f5c2 that changes `.vscode/settings.json` to insert a newline at the end of any file.